operators/pkg/forge/labels.go

```diff
@@ -93,3 +97,16 @@ func persistentLabelValue(environmentList []clv1alpha2.Environment) string {
 	}
 	return strconv.FormatBool(false)
 }
+
+// SandboxLabels receives in input a set of labels and returns the updated set depending on the specified template,
+// along with a boolean value indicating whether an update should be performed.
+func SandboxLabels(labels map[string]string, sbName, targetLabelKey, targetLabelValue string) (map[string]string, bool) {
+	labels = deepCopyLabels(labels)
+	update := false
+
+	update = updateLabel(labels, labelManagedByKey, tnlabelManagedByValue) || update
+	update = updateLabel(labels, labelTypeKey, sblabelManagedByValue) || update
+	update = updateLabel(labels, labelNameKey, sbName) || update
+
+	return labels, update
+}
```

Comment:
* Always ensure comments are valid when copied from other functions
* Please, move this function below `InstanceObjectLabels`, and rename it to `SandboxObjectLabels` for consistency.
* The *update* part is superfluous and can be dropped. You can assign the values directly, as in the `InstanceObjectLabels` function.
* The `target*` parameters are unused and can be dropped.
* The *labelNameKey* label can be dropped, and replaced with the *labelTenantKey* one